CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Chef v3.9.4_7.2.0
+
+### Added
+
+- Update to Wazuh version 3.9.4_7.2.0
+- Added the choice to register an agent or not ([rshad](https://github.com/rshad)) [PR#50](https://github.com/wazuh/wazuh-chef/pull/50)
+
+- Added Filebeat's module installation to the installation tasks ([rshad](https://github.com/rshad)) [PR#55](https://github.com/wazuh/wazuh-chef/pull/55)
+
+### Changed
+
+- Upgraded wazuh_api to accept Vault, Data Bags or none of them to configure Wazuh API ([@jm404] (https://github.com/jm404)) [PR#47](https://github.com/wazuh/wazuh-chef/pull/47)
```

Comment:
My name and link are not well-formatted, please have a look at it

![image](https://user-images.githubusercontent.com/46649922/63002213-acbc8d80-be75-11e9-846d-c4b04175c2a9.png)

Reply:
Hi @jm404,

Sorry, this is was a typo.

Here you have the [correction](https://github.com/wazuh/wazuh-chef/commit/60696b0ad99c51ecff81447a6b6dca80e3b78dcd).

Kind regards,

Rshad